easybuild/easyconfigs/r/ReFrame/ReFrame-3.0.eb

```diff
@@ -0,0 +1,93 @@
+easyblock = 'PythonPackage'
```

Comment:
@teojgo You should use `PythonBundle`, and install ReFrame itself as an extension too (last in `exts_list`).

With the current approach, you're installing ReFrame first, which will result auto-downloading all required dependencies from PyPI, and then afterwards specific versions of the dependencies will be installed again as extensions.

That's sort of backwards...

Reply:
I wanted to avoid PythonBundle since I don't want to rely on a Python module. Is there a way to tell EasyBuild to use the system python?